spec/features/atom/issues_spec.rb

```diff
@@ -13,10 +13,10 @@
         login_with user
         visit project_issues_path(project, :atom)
 
-        response_headers['Content-Type'].should have_content("application/atom+xml")
-        body.should have_selector("title", text: "#{project.name} issues")
-        body.should have_selector("author email", text: issue.author_email)
-        body.should have_selector("entry summary", text: issue.title)
+        expect(response_headers['Content-Type']).to have_content("application/atom+xml")
+        expect(body).to have_selector("title", text: "#{project.name} issues")
+        expect(body).to have_selector("author email", text: issue.author_email)
+        expect(body).to have_selector("entry summary", text: issue.title)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.